README.md

```diff
@@ -601,9 +601,11 @@ prod
 
 ```javascript
 const args = {
+  "ordered": "1",  
```

Comment:
I was unable to get ordered to work using a string `"1"`. This works for me when using an integer  `1`.